pycbc/events/stat.py

```diff
@@ -1847,6 +1873,16 @@ def assign_kdes(self, kname):
         with h5py.File(self.files[kname + '-kde_file'], 'r') as kde_file:
             self.kde_by_tid[kname + '_kdevals'] = kde_file['data_kde'][:]
 
+    def kde_logsignalrate(self):
```

Comment:
I would rather call this kde_ratio .. it is not a 'signal rate' particularly

One could think of it as a rate of signals-per-template, but basically it's just the ratio of the kdes evaluated at the templates